math/minuit2/inc/Minuit2/GradientCalculator.h

```diff
@@ -35,6 +35,10 @@ class GradientCalculator {
 
    virtual bool G2(const MinimumParameters &, MnAlgebraicVector &) const { return false;}
 
+   static bool DoParallelOMP(bool doParallel = true);
```

Comment:
```suggestion
   static bool SetParallelOMP(bool doParallel = true);
```
...or `EnableParallelOMP()` maybe...

The problem with the `Do*` prefix is that it's not clear from the name whether this is a setter or a getter, and more importantly this name breaks a convention that is everywhere in the math code: functions starting with `DoFoo` are usually implementation details that are called by the public `Foo` function. For example here in the fitter:

https://github.com/root-project/root/blob/master/math/mathcore/inc/Fit/Fitter.h#L174

So maybe it's not too late to change the name?